hummingbot/connector/exchange/ascend_ex/ascend_ex_api_user_stream_data_source.py

```diff
@@ -68,8 +72,8 @@ async def listen_for_user_stream(self, ev_loop: asyncio.BaseEventLoop, output: a
         ws = None
         while True:
             try:
-                headers = self._ascend_ex_auth.get_headers(CONSTANTS.INFO_PATH_URL)
-                rest_assistant = await self._get_rest_assistant()
+                headers = self._ascend_ex_auth.get_auth_headers(CONSTANTS.INFO_PATH_URL)
```

Comment:
The authentication headers should be automatically included in the request by the logic inside the RESTAssistant (interacting with the auth object)